grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -78,6 +79,23 @@ class GrabbitConfiguration {
         )
     }
 
+    private
+    static Collection<String> nonEmptyCollection(Collection<String> excludePaths, ConfigurationException.Builder errorBuilder) {
+        String prePattern = "^(/|\\./|\\|\\\\).*\$";
+        String postPattern = ".*(/|\\./|\\|\\\\)\$";
+        if (excludePaths == null || excludePaths.isEmpty())
+            return excludePaths
```

Comment:
Where this is called, you're making sure this condition would never be triggered. Better would be to do the empty collection here.

So the calling code would be
```groovy
def excludePaths = nonEmptyCollection(config["excludePaths"], errorBuilder)
```
and then this condition would be
```groovy
if (excludePaths == null || excludePaths.isEmpty())
    return Collections.EMPTY_LIST
```